ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  Link,
+  LinkifiedText,
+  Text,
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Link href={linkAttachment.url} unstyled>
+          <Card
+            noPadding
+          >
+            <div className={style['link-attachment-content']}>
+              <div className={style['link-attachment-image']}>
+                <Image
+                  height={'100%'}
```

Comment:
I think you can remove this rule. At the moment this seems to lock the image as a square.